### PR TITLE
Configuring policy package

### PR DESCRIPTION
**Pull request by dynamic-entropy**

### CODE REVIEW

File: apps/base/cms-rucio-common.yaml [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+  tag: release-1.19.5 #NOTE: Is this rucio image version?
 
 config:
     accounts:
         special_accounts: "tier0"
     policy:
-        permission: "cms"
-        schema: "cms"
+        package: "CMSRucioPolicy"
+        # permission: "cms"
```
Comment (ericvaandering) [inline apps/base/cms-rucio-common.yaml:9]: Please get rid of commented out lines. This leave code and configs very hard to follow after a time.

**Review by ericvaandering — CHANGES_REQUESTED** [review]

File: apps/base/cms-rucio-common.yaml [hunk 2]
```diff
@@ -1,15 +1,16 @@
 image:
-  tag: release-1.19.5
+  tag: release-1.19.5 #NOTE: Is this rucio image version?
```
[the remark this answers is not in the document]
Comment (dynamic-entropy) [reply apps/base/cms-rucio-common.yaml:2]: Ah, thank you. 
Why don't we remove these values then? is it "required" here too?
